### PR TITLE
Adds gasLimitEstimate for bulk payout

### PR DESCRIPTION
**Pull request by SaptakS**
##### Description

Fixes the error in the bulk payout page where gasLimit came to be 0.

##### Checklist

- [x] linter status: 100% pass
- [x] changes don't break existing behavior
- [x] commit message follows [commit guidelines](https://github.com/gitcoinco/web/blob/master/docs/CONTRIBUTING.md#step-4-commit)

##### Affected core subsystem(s)
Gas Settings for bulk payout page fixed

### CODE REVIEW

**Comment by owocki** [comment]
this is a good change to have, but its not the root cause of the bug i reported.

> Fixes the error in the bulk payout page where gasLimit came to be 0.

http://bits.owocki.com/441f3K1M1Z1d/Screen%20Shot%202018-07-17%20at%2010.06.33%20AM.png

gas limit was not zero, gas *price* was

**Comment by SaptakS** [comment]
@owocki I just checked. Seems for me, making this change fixes the gasPrice computation as well in local. Did you check it by pulling locally?

File: app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js [hunk 1]
```diff
@@ -23,6 +23,10 @@ var gas_amount = function(page_url) {
   if (page_url.indexOf('issue/cancel') != -1) {
     gasLimitEstimate = 67327;
   }
+  // bulk payout
+  if (page_url.indexOf('issue/payout') != -1) {
+    gasLimitEstimate = 103915;
+  }
   // tip
   if (page_url.indexOf('tip/send') != -1) {
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js]: Can we make them as `else if` conditions ?

**Comment by owocki** [comment]
> Did you check it by pulling locally?

no, thanks for testing.  

i must have been mistaken; maybe somehow the two are coupled together within Metamask's codebase and one being missing causes the other to be missing too.

File: app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js (hunk 1 above)
Comment (SaptakS) [reply app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js]: True. Will refactor that.

Comment (SaptakS) [reply app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js]: Done.

File: app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js [hunk 2]
```diff
@@ -6,29 +6,19 @@ var gas_amount = function(page_url) {
...
-  }
-  // new kill funding page
-  if (page_url.indexOf('issue/cancel') != -1) {
+  } else if (page_url.indexOf('issue/cancel') != -1) { // new kill funding page
     gasLimitEstimate = 67327;
-  }
-  // tip
-  if (page_url.indexOf('tip/send') != -1) {
+  } else if (page_url.indexOf('issue/payout') != -1) { // bulk payout
+    gasLimitEstimate = 103915;
```
Comment (thelostone-mc) [inline app/assets/v2/js/pages/shared_bounty_mutation_estimate_gas.js]: should we club this along with `(page_url.indexOf('issue/accept') != -1)` aka [L13](https://github.com/gitcoinco/web/pull/1776/files#diff-15e269c947d7b3b6ece821641f15ada9R13)

@SaptakS @owocki

**Pull request merged**